packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -128,6 +128,50 @@ class Neo4jGraphQL {
         await assertIndexesAndConstraints({ driver, driverConfig, nodes: this.nodes, options: input.options });
     }
 
+    private wrapResolvers(resolvers: IResolvers, { schema }: { schema: GraphQLSchema }) {
```

Comment:
**nitpick (non-blocking)**: Maybe we could move these functions to the bottom?
Usually, it is slightly easier to read functions in a class from more abstract to less abstract (kind of like articles are composed). So, first, we read `generateSchema` at the top, and then, if we need to, we keep going to the functions that are used by that one